test/rules/sort-imports.test.ts

```
@@ -4468,183 +4307,68 @@ describe('sort-imports', () => {
-      await invalid({
-        errors: [
-          {
-            data: {
-              right: '@a/a1',
-              left: 't',
-            },
-            messageId: 'unexpectedImportsOrder',
-          },
+    it('handles hash symbol in internal patterns correctly', async () => {
+      await valid({
+        code: dedent`
```

ESLint

[error] Unsafe use of a(n) `error` type typed template tag.

(typescript/no-unsafe-call)